src/test/java/com/jcabi/w3c/DefaultHtmlValidatorTest.java

```diff
@@ -78,12 +72,41 @@ public void validatesHtmlDocument() throws Exception {
     }
 
     /**
+     * Test if {@link DefaultHtmlValidator} validades invalid html.
+     * @throws Exception If something goes wrong inside
+     */
+    @Test
+    public void validateInvalidHtml() throws Exception {
+        final MkContainer container = new MkGrizzlyContainer().next(
+                        new MkAnswer.Simple(this.invalidHtmlResponse())
+        ).start();
+        final Validator validator = new DefaultHtmlValidator(container.home());
+        final ValidationResponse response = validator
+            .validate("this is an invalid html");
+        container.stop();
+        MatcherAssert.assertThat(
+                        "Validity must be invalid!",
+                        !response.valid()
+        );
+        MatcherAssert.assertThat(
```

Comment:
@jrdalpra 
```
MatcherAssert.assertThat(
                        "Must has at least one error",
                        response.errors(), this.withoutDefects()
        );
```
should be 
```
MatcherAssert.assertThat(
    "Must has at least one error",
    response.errors(), 
    this.withoutDefects()
);
```

same below

I wonder why checkstyle doesn't report about it